m4/sage_spkg_configure.m4

```diff
@@ -185,22 +185,40 @@ AC_DEFUN([SAGE_SPKG_CONFIGURE], [
 
 # SYNOPSIS
 #
-#   SAGE_SPKG_DEPCHECK(PACKAGE-DEPENDENCIES, FURTHER-CHECK)
-#                                 $1              $2
+#   SAGE_SPKG_DEPCHECK(PACKAGE-DEPENDENCIES, FURTHER-CHECK, [FAIL= [sage_spkg_install_]SPKG_NAME=yes])
```

Comment:
The documentation needs more work, so that we don't have to repeat this type of conversation the next time.

`FAIL` is unclear -- what is failing. This needs clearer terminology.

Reply:
please check the new version